sdk/tests/deqp/functional/gles3/es3fNegativeShaderApiTests.js

```diff
@@ -485,11 +485,11 @@ goog.scope(function() {
 
                 bufferedLogToConsole('gl.INVALID_OPERATION is generated if the size of the uniform variable declared in the shader does not match the size indicated by the glUniform command.');
                 gl.useProgram(program.getProgram());
-                gl.uniform1fv(vec4_v, data);
+                gl.uniform1fv(vec4_v, new Float32Array(1));
```

Comment:
Zhenyao, if we set the size to 12, then there are two errors: 
1) the size of the uniform variable declared in the shader (with 4 elements) does not match the size indicated by the glUniform command (uniform1fv, means 1 element). 
2) count is greater than 1 (12 / 1 = 12), but the variable declared in the shader is not an array. 

If the data is Float32Array(1) and Float32Array(2), etc as I set here, the count will be 1, there would be no 2) error.